README.md

```
@@ -129,6 +129,43 @@ envVars:
+| `monitoring.jmx.enabled` | Enable JMX monitoring | `true` |
+| `monitoring.jmx.exporter.image` | JMX exporter image | `bitnami/jmx-exporter:1.1.0` |
+| `monitoring.jmx.exporter.pullPolicy` | JMX exporter image pull policy | `IfNotPresent` |
+| `monitoring.jmx.exporter.hostPort` | JMX host and port | `127.0.0.1:1098` |
+| `monitoring.jmx.exporter.lowercaseOutputName` | Convert output metric names to lowercase | `true` |
+| `monitoring.jmx.exporter.lowercaseOutputLabelNames` | Convert output label names to lowercase | `true` |
+
+### JMX Monitoring
+
+The chart provides JMX monitoring capabilities for ActiveMQ Artemis. JMX monitoring allows you to:
+
+- Monitor broker performance metrics
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)